lib/internal/bootstrap/node.js

```diff
@@ -377,14 +376,8 @@ function setupProcessObject() {
     configurable: false,
     value: 'process'
   });
-  // Make process globally available to users by putting it on the global proxy
-  ObjectDefineProperty(globalThis, 'process', {
-    __proto__: null,
-    value: process,
-    enumerable: false,
-    writable: true,
-    configurable: true
-  });
+
+  // globalThis.process is been installed at `internal/bootstrap/pre_execution.js`.
```

Comment:
```suggestion
  // globalThis.process will be installed at `internal/bootstrap/pre_execution.js`.
```

Reply:
But why are we removing these anyway? Removing these results in a breaking change for the embedders, which is why the tests have to be updated.

Reply:
I was reviewing the installation point of `globalThis.process` and `globalThis.Buffer`. It can be more intuitive to install them once rather than duplicate the installation.

It is true that removing can cause breaking changes on embedders with custom entry points. What do you think about moving the actual installation here?

Reply:
I think the reason that they are overriden in pre_execution is that we thought we were going to make this optional for ESM (probably behind flags), which didn't seem to happen...

Reply:
Yeah, that's true. From what I could tell, the current shape of the property descriptor is not going to be changed in anytime soon.